test/e2e/ginkgo/ginkgo_suite_test.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2023 The KubeStellar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"flag"
+	"testing"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	ocmWorkClient "open-cluster-management.io/api/client/work/clientset/versioned"
+
+	"k8s.io/client-go/kubernetes"
+
+	ksClient "github.com/kubestellar/kubestellar/pkg/generated/clientset/versioned"
+	"github.com/kubestellar/kubestellar/test/util"
+)
+
+func TestGinkgo(t *testing.T) {
+	gomega.SetDefaultEventuallyTimeout(time.Minute)
+	gomega.RegisterFailHandler(ginkgo.Fail)
+	ginkgo.RunSpecs(t, "Kubestellar testing")
+}
+
+var (
+	ctx          context.Context
+	coreCluster  *kubernetes.Clientset
+	wds          *kubernetes.Clientset
+	ksWds        *ksClient.Clientset
+	wec1         *kubernetes.Clientset
+	wec2         *kubernetes.Clientset
+	ocmWorkImbs  *ocmWorkClient.Clientset
```

Comment:
We are going with "ITS" rather than "IMBS". Yes, some other stuff is outdated too.

FYI: I would not mind seeing only "work" here rather tan "ocmWork".

Reply:
Oh I didn't know about ITS.  I humbly suggest change in name is done all at once, i.e. search and replace all IMBS into ITS.

Reply:
We already have a mixture of ITS and IMBS. I think new work should use the new name; cleaning up outdated stuff can be done separately.

Reply:
Ok.  I scrubbed the term IMBS and replaced it with ITS.